guideline/cfh.md

```diff
@@ -83,8 +85,7 @@ These are the requirements that an on-site team will sign-up to when submitting
 a proposal. Changes to these requirements are possible, but they must be signed
 off by the PAC before they can be put in place.
 
-1. The conference will be financially and legally run by an entity registered
-   at the hosting country and for which the on-site team is part of.
```

Comment:
The choice of "part of" instead of "members of" leaves enough room for the relation between the organizing team and the representing entity.